packages/field-plugin/src/messaging/pluginMessage/containerToPluginMessage/LoadedMessage.ts

```diff
@@ -27,7 +27,7 @@ export type LoadedMessage = MessageToPlugin<'loaded'> & {
 }
 
 // TODO full implementation of validation
-export const isLoadedMessage = (data: unknown): data is LoadedMessage =>
+export const isLoadedMessage = (data: unknown): boolean =>
```

Comment:
This is going to disable the [type narrowing](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates)

Reply:
This probably explains my warning below 🤔

Reply:
They are no longer appearing after the last commit 🙏